packages/component-library/src/components/form/mt-button/mt-button.vue

```diff
@@ -22,7 +22,13 @@
   >
     <mt-loader v-if="isLoading" size="16px" class="mt-button__loader" />
     <span class="mt-button__content" :class="contentVisibilityClass">
-      <slot />
+      <span v-if="$slots.iconFront" class="mt-button__icon mt-button__icon--front">
+        <slot name="iconFront" />
+      </span>
+      <slot></slot>
+      <span v-if="$slots.iconBack" class="mt-button__icon mt-button__icon--back">
```

Comment:
Place center the contents when the button is in it's ["block mode"](https://meteor-component-library-git-update-m-c7f47a-shopware-frontends.vercel.app/?path=%2Fstory%2Finteraction-tests-form-mt-button--test-default-slot&args=iconFront%3Asdf%3BiconBack%3Aasdf%3Bsize%3Alarge%3Bblock%3A%21true)

Reply:
Added a flex property to the block class to keep everything centred